app/webpacker/styles/application.scss

```diff
@@ -1,5 +1,7 @@
 @charset 'UTF-8';
 
+$govuk-font-family: Sans-serif;
```

Comment:
The rest of the site is using the 'system font stack' - I think this should be doing the same for consistency.

Since that may change in the future, I think the font family should be set in a variable before this govuk one, and the govuk one derives from the GiT one? Likewise anything else in the SCSS which assigns the system font stack should instead use the GiT font variable?

Reply:
Where are we defining a font stack? I couldn't find any reference to one.

```
$ ag font-family

app/webpacker/styles/layout.scss
2:    font-family: sans-serif;

app/webpacker/styles/application.scss
3: $govuk-font-family: Sans-serif;

app/webpacker/styles/text.scss
2:    font-family: sans-serif;
```

Reply:
Ok, I added `$git-font-family` variable that's defined in `text.scss` and used that to override the hard-coded `sans-serif` rules.